migrations/2022-06-21-123144_language-tags/up.sql

```diff
@@ -0,0 +1,195 @@
+create table language (
+  id serial primary key,
+  code varchar(3),
+  name text
+);
+
+alter table local_user rename column lang to interface_language;
```

Comment:
`interface_language_id`

Reply:
No, we should leave this separete from discussion languages, because the valid values are different. For example it has a default value of `browser`, which makes no sense for discussion language.

Reply:
Ah ok. This column isn't joined to anything anyway.